Multiple productions and venues

randycoulman opened the pull request:
Allow specifying multiple productions and venues.

Editing:
![screen shot 2015-09-10 at 3 04 01 pm](https://cloud.githubusercontent.com/assets/1406203/9802950/fdefee76-57d2-11e5-913d-83575dd74001.png)

Browsing:
![screen shot 2015-09-10 at 3 43 06 pm](https://cloud.githubusercontent.com/assets/1406203/9802949/fded50ee-57d2-11e5-81e1-bcd31c3aeab6.png)

Commit:
Allow multiple productions and venues

I believe I’ve caught everywhere that needs to be changed.  Everything
seems to be working properly.

Commit:
Show names, not ids in file browse view

Commit:
Only show/edit id fields, not name fields

Commit:
Provide better labels when editing files

Commit:
Reformat to satisfy Hound

Commit:
More Hound cleanups

Commit:
Hound yet again

Commit:
Tweak Ruby style enforced by Hound

We want method cascades to have a leading `.`, not a trailing one.
This is especially true for collection pipelines.

randycoulman committed:
Merge branch 'master' into features/multiple_productions_and_venues

Conflicts:
	.hound.yml

randycoulman committed:
Remove duplicate line

randycoulman committed:
Clean up show views

Add raw delegators for the name attributes to the
GenericFilesPresenter.  This allows us to use those in the view
templates rather than having to go look up the production credits
records to access the names from there.

seanculver commented on app/controllers/catalog_controller.rb:
```diff
@@ -1,11 +1,12 @@
 # -*- coding: utf-8 -*-
```
I'd rather just ignore this file in the rubocop file then uglify it. Thoughts?

randycoulman committed:
Unwrap the long lines in CatalogController

General team consensus is that the longer lines are more readable.  We
could do some additional metaprogramming, but don’t want to deviate
from the generated content more than necessary.

Modify the hound configuration to ignore the line-length checking on
this file.

seanculver commented:
:+1:
